src/context.ts

```diff
@@ -58,6 +58,11 @@ export interface Context<S extends ConfigurationSubject, C extends Settings> {
      * Forces the currently displayed tooltip, if any, to update its contents.
      */
     forceUpdateTooltip(): void
+
+    /**
+     * Experimental client capabilities that haven't been merged in the protocol yet.
+     */
+    experimentalClientCapabilities?: any
```

Comment:
`experimentalClientCapabilities` is now in `Context`

Reply:
There is no requirement for these experimental client capabilities to eventually be merged into the protocol spec. Also, it would be helpful to describe how these are used:

> Experimental capabilities implemented by the client (that are not defined by the CXP specification). These capabilities are passed verbatim to extensions in the `initialize` request's `capabilities.experimental` property.

Reply:
That's a much nicer doc string :+1: